src/components/RichText/_text.scss

```diff
@@ -0,0 +1,80 @@
+// ----------------------------------
+// UI Components
+// Text | TextSnippet
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.cc-text {
```

Comment:
I created [this file](https://github.com/wellcometrust/corporate-components/pull/273/files#diff-c8dfa52107486fa946af088a312fae3b) as part of a previous PR.

I created it to cover style issues where we don't (or in future _wont_) have control over the markup (e.g. from an RTE).

I feel like we could drop the file I created and use the RichText component you've created to render all instances of rich text... thoughts?

Reply:
This file is actually a duplicate of (Text/_text.scss) which I've now removed.

RichText could be used for everything but one thing to bear in mind is that there may be different styles in place for different components

e.g. I can see in your _rte.scss you have bottom margins removed from all RTE child elements which would be at odds with _text.css

Happy to standardise though and we can work towards that in future PRs.